app/src/main/res/layout/activity_music_temporary.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MusicTemporary">
+
+    <TextView
+        android:id="@+id/textWelcome"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="112dp"
+        android:layout_marginTop="208dp"
+        android:layout_marginEnd="111dp"
+        android:layout_marginBottom="23dp"
+        android:text="Enter a Music Name:"
```

Comment:
consider using @string instead of hardcoded string

Reply:
Yes, but as it is a temporary activity, I didn't put it in the String.xml otherwise It will flood it with not important value :)